mesheryctl/internal/cli/root/system/start.go

```diff
@@ -372,14 +372,14 @@ func start() error {
 		}
 
 		if err != nil {
-			return err
-		}
-
-		currCtx.Endpoint = utils.EndpointProtocol + "://" + endpoint.External.Address + ":" + strconv.Itoa(int(endpoint.External.Port))
+			log.Info("service endpoint not discovered. Meshery deployed on default endpoint")
+		} else {
+			currCtx.Endpoint = utils.EndpointProtocol + "://" + endpoint.External.Address + ":" + strconv.Itoa(int(endpoint.External.Port))
```

Comment:
To a new user, what is a "default endpoint"? It would be confusing and the user may not know what to do next.

// @hexxdump Suggestions?